index.js

```diff
@@ -12,6 +13,15 @@ module.exports = function (options) {
 			return cb();
 		}
 
+		if (file.data) {
+  			//options = file.data;
+  			//custom objext.index for prototype
+  			options = file.data.results[0];
```

Comment:
Why is it looking for a results array on file.data? 

The gulp-data example does not work.

Eg - this **should** work with gulp data:

```
gulp.src('path/to/templates/**/*.nunj')
  .pipe( data( function(file){
    return { example: 'data for ' + file.relative };
  }) )
  .pipe( nunjucksRender() )
  .pipe( gulp.dest('path/to/output')
;
```